### DISCUSSION TITLE
How to get Total Memory usage for a query from Velox

### DISCUSSION THREAD

**Discussion by GuChangyang** [facebookincubator/velox#7472] [Q&A]
Is it possible to get the total memory allocated of a query from velox?

Thank you

**Comment by xumingming** [comment]
Velox is a `single node` execution engine, you can only get the memory usage in single node, all tasks of the same query shall share the same QueryCtx, you can get the memory usage in single node through: `QueryCtx->pool_->currentBytes()`.
